src/components/crossDevice/CrossDeviceSubmit/style.scss

```diff
@@ -1,9 +1,9 @@
-@import (less) "../../Theme/constants.css";
+@import '../Theme/constants';
```

Comment:
I think this needs to be `'../../Theme/constants';` due to the level of the folders